lib/presentation/models/word_cloud/word_cloud_data.dart

```diff
@@ -0,0 +1,78 @@
+import 'package:flutter/painting.dart';
+import 'dart:ui';
+
+class FlutterHashtag {
+  const FlutterHashtag(
+    this.hashtag,
+    this.color,
+    this.size,
+    this.rotated,
+  );
+  final String hashtag;
+  final Color color;
+  final int size;
+  final bool rotated;
+}
+
+class FlutterColors {
```

Comment:
Same here, rename it to`KeywordColors`. Could you also move that class to a new seperate file: `core/theme/keyword.dart`

Reply:
Addressed in commit f4f228145ac6d4a9c0ea067822b119dbcafb5650